.travis.yml

```diff
@@ -26,6 +26,8 @@ matrix:
     - php: 5.4
       env: COVERALLS=1 DEFAULT=0 DB=mysql db_dsn='mysql://travis@0.0.0.0/cakephp_test'
 
+    - php: 7.0
```

Comment:
Sorry i should have mentioned this also earlier but 7.0 should be added at the top under `php:`